integrations/argocd/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Dockerfile with a script to use terrascan's validating webhook
+# configured in the kubernetes cluster, to scan a repo for violations
+FROM alpine:3.12.0
+
+#curl to send request to terrascan validating webhook
+RUN apk add --no-cache curl 
+
+WORKDIR /home/terrascan
+
+RUN mkdir bin
+
+COPY scripts/argocd-terrascan-remote-scan.sh  bin/terrascan-remote-scan.sh
+
+# create non root user
+RUN addgroup --gid 101 terrascan && \
```

Comment:
Can we combine all the RUN commands? It is just good practice, reduces docker image layers and size?